examples/spark-pi-prometheus.yaml

```
@@ -22,28 +22,28 @@ metadata:
 spec:
   type: Scala
   mode: cluster
-  image: gcr.io/spark-operator/spark:v3.1.1-gcs-prometheus
+  image: spark:3.5.2-gcs-prometheus
```

```suggestion
  image: {IMAGE_REGISTRY}/{IMAGE_REPOSITORY}/spark:3.5.2-gcs-prometheus
```

Since the image `spark:3.5.2-gcs-prometheus` does not actually exist, we might consider adding `{IMAGE_REGISTRY}/{IMAGE_REPOSITORY}` as a placeholder to remind users that they need to build and push the image to their own image registry.

---

I think it's a good suggestion.
Added.